docs/plugins/cloud-gce.asciidoc

```diff
@@ -104,97 +106,123 @@ Table of resources:
 +-----------+--------------+-------+---------+--------------+----------------+----------------+----------------+---------+----------------+
 | myesnode1 | f1-micro     |       | default | 10.240.20.57 | 192.158.29.199 | boot-myesnode1 | europe-west1-a | RUNNING |                |
 +-----------+--------------+-------+---------+--------------+----------------+----------------+----------------+---------+----------------+
-```
+--------------------------------------------------
 
 You can now connect to your instance:
 
-```
+[source,sh]
+--------------------------------------------------
 # Connect using google cloud SDK
 gcloud compute ssh myesnode1 --zone europe-west1-a
 
 # Or using SSH with external IP address
 ssh -i ~/.ssh/google_compute_engine 192.158.29.199
-```
+--------------------------------------------------
 
-*Note Regarding Service Account Permissions*
+[IMPORTANT]
+.Service Account Permissions
+==============================================
 
 It's important when creating an instance that the correct permissions are set. At a minimum, you must ensure you have:
```

Comment:
This may not apply, but has any of the packaging/plugin work changed what we expect for permissions when installing/using the cloud plugins?